js/main_nav.js

```diff
@@ -0,0 +1,26 @@
+let overlay = document.getElementById("main-nav");
+
+let navBurguer = document.getElementById("main-nav__burguer");
+
+let overlayMenu = document.getElementById("main-nav__menu-items");
+
+navBurguer.addEventListener("click", function () {
+	this.classList.toggle("burguer-animation");
+});
+
+navBurguer.addEventListener("click", function () {
+	overlay.classList.toggle("main-nav-overlay__animation");
+});
+
+navBurguer.addEventListener("click", function () {
+	overlayMenu.classList.toggle("main-nav__menu-items__animation");
+});
+
+window.addEventListener("scroll", function () {
+	if (window.scrollY >= 80) {
```

Comment:
A ternary operator could reduce this block of code.

Reply:
The knowledge has no limits, done.